api/envoy/config/filter/http/gzip/v2/gzip.proto

```diff
@@ -95,4 +95,9 @@ message Gzip {
   // the fields `content_length`, `content_type`, `disable_on_etag_header` and
   // `remove_accept_encoding_header` are ignored.
   compressor.v2.Compressor compressor = 10;
+
+  // Value for Zlib's next output buffer. If not set, defaults to 4096.
+  // See https://www.zlib.net/manual.html for more details. Also see
+  // https://github.com/envoyproxy/envoy/issues/8448 for context on this filter's performance.
+  google.protobuf.UInt64Value chunk_size = 11 [(validate.rules).uint64 = {lte: 65536 gte: 4096}];
```

Comment:
nit: I'd swap the `lte` and `gte` constraints to resemble a conventional interval.

Reply:
everywhere else the convention seems to be `{lte: ... gte: ...}` (e.g.: `git grep `{gte` gives nothing with `lte` at the end). So let's stick to the convention.

Reply:
Alright. nvm then